mysql-test/suite/innodb/t/rename_table.test

```diff
@@ -30,16 +30,21 @@ DROP DATABASE abc_def;
 
 DROP DATABASE abc_def2;
 
-call mtr.add_suppression("InnoDB: (Operating system error|The error means|Cannot rename file)");
+call mtr.add_suppression("InnoDB: Cannot rename '.*/test/t1.ibd' to '.*non_existing_db.*' because the target schema directory doesn't exist");
```

Comment:
For some reason this suppression does not work on Microsoft Windows. Also the test `mariabackup.rename_during_backup` fails due to a similar reason.

Reply:
Fixed.